frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1007,6 +1007,15 @@ export class WorkflowEditor extends BtrixElement {
         msg(`If checked, the crawler will visit pages one link away.`),
         false,
       )}
+      ${inputCol(html`
+        <sl-checkbox
+          name="failOnContentCheck"
+          ?checked=${this.formState.failOnContentCheck}
+        >
+          ${msg("Fail crawl if not logged in")}
+        </sl-checkbox>
+      `)}
+      ${this.renderHelpTextCol(infoTextFor["failOnContentCheck"])}
```

Comment:
This will fix alignment:
```suggestion
      ${this.renderHelpTextCol(infoTextFor["failOnContentCheck"], false)}
```